app/boards/shields/murphpad/murphpad.zmk.yml

```
@@ -0,0 +1,14 @@
+file_format: "1"
+id: murphpad
+name: MurphPad
+type: shield
+url: https://mechwild.com/product/murphpad/
+requires:
+    - pro_micro
+exposes:
+    - i2c_oled
```

```suggestion
requires: [pro_micro]
exposes: [i2c_oled]
```
Metadata validation is failing, and I believe it needs this change to fix